### DISCUSSION TITLE
Draft Project Checklist: "General workflow, as well as alternatives"

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#73] [Q&A]
The list of stuff for our draft project includes the following bullet point
" Your readme file needs to describe the process and the general workflow, **as well as alternatives**"

What is meant by "as well as alternatives" ?

**Comment by sr320** [comment]
What is a different way of getting same general outcome. 
In your case you might be using kallisto to get count information.  What other programs could be used?
